src/connector.js

```diff
@@ -148,14 +152,14 @@ if ( !resultTemplateHTML ) {
 			`<h3><a class="result-link" href="%[result.clickUri]" data-dtm-srchlnknm="%[index]">%[result.title]</a></h3> 
 			<ul class="context-labels"><li>%[result.raw.author]</li></ul> 
 			<ol class="location"><li>%[result.printableUri]</li></ol> 
-			<p><time datetime="%[short-date-en]" class="text-muted">%[long-date-en]</time> - %[highlightedExcerpt]</p>`;
+			<p><time datetime="%[short-date-fr]" class="text-muted">%[long-date-fr]</time> - %[highlightedExcerpt]</p>`;
 	}
 	else {
 		resultTemplateHTML = 
 			`<h3><a class="result-link" href="%[result.clickUri]" data-dtm-srchlnknm="%[index]">%[result.title]</a></h3> 
 			<ul class="context-labels"><li>%[result.raw.author]</li></ul> 
 			<ol class="location"><li>%[result.printableUri]</li></ol> 
```

Comment:
Please make the Breadcrumb the default for the link line in green, instead of the URL. So please replace `result.printableUri` by whatever is needed for the breadcrumb. And please ensure the if there is no breadcrumb for a source, that the fallback is the printable URL and not NOTHING. Thank you :)